src/Data/Bool/Properties.agda

```diff
@@ -29,7 +29,7 @@ open import Relation.Binary.Definitions
 open import Relation.Binary.PropositionalEquality.Core
 open import Relation.Binary.PropositionalEquality.Properties
 open import Relation.Nullary.Reflects using (ofʸ; ofⁿ)
```

Comment:
This import can be removed entirely; it is only required by the definition of `T?`.

Reply:
Nice spot!